TM1py/Services/ElementService.py

```diff
@@ -275,3 +276,21 @@ def get_members(element):
 
         get_members(consolidation_tree)
         return members
+
+    def execute_set_mdx(self, mdx, top=10000):
```

Comment:
In order to be compliant with the rest of TM1py I think it would be nice to set the default value for the argument to `None`.

Reply:
Fixed in the latest commit